doc/nrf/gs_recommended_versions.rst

```diff
@@ -196,8 +196,6 @@ Building documentation
      - |sphinxcontrib-mscgen_ver|
    * - breathe
      - |breathe_ver|
-   * - docutils
-     - |docutils_ver|
    * - sphinx
      - |sphinx_ver|
    * - sphinx_rtd_theme
```

Comment:
This is not a direct dependency anymore, we need `sphinx-ncs-theme` here.

Reply:
Which version do we need?

Reply:
latest, >=0.6.2